src/coreclr/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -2881,6 +2896,16 @@ void CodeGen::genAvxFamilyIntrinsic(GenTreeHWIntrinsic* node, insOpts instOption
         case NI_AVX512BW_VL_ConvertToVector128ByteWithSaturation:
         case NI_AVX512BW_VL_ConvertToVector128SByte:
         case NI_AVX512BW_VL_ConvertToVector128SByteWithSaturation:
+        case NI_AVX10v1_ConvertToVector128Byte:
+        case NI_AVX10v1_ConvertToVector128ByteWithSaturation:
+        case NI_AVX10v1_ConvertToVector128Int16:
+        case NI_AVX10v1_ConvertToVector128Int16WithSaturation:
+        case NI_AVX10v1_ConvertToVector128Int32:
+        case NI_AVX10v1_ConvertToVector128Int32WithSaturation:
+        case NI_AVX10v1_ConvertToVector128SByte:
+        case NI_AVX10v1_ConvertToVector128SByteWithSaturation:
+        case NI_AVX10v1_ConvertToVector128UInt16:
+        case NI_AVX10v1_ConvertToVector128UInt16WithSaturation:
```

Comment:
I think we're missing `NI_AVX10v1_X64_ConvertScalarToVector128*` handling somewhere around here where `NI_AVX512F_X64_ConvertScalarToVector128*` is being handled

Reply:
same as discussion here https://github.com/dotnet/runtime/pull/101938#discussion_r1607436860

Reply:
Resolved here https://github.com/dotnet/runtime/pull/101938#discussion_r1609255657